### DISCUSSION TITLE
How to get to work in front end?

### DISCUSSION THREAD

**Discussion by buttonsbond** [Quantum-Manager/tracker#99] [Q&A]
Love the quantum manager - however I can't seem to get it to work for a logged in user from the front end, nor can I get it to launch when I click the 'image' icon in JCE editor, only if I use the image button, and then only when logged into the back end.

I also saw in the documentation that you can get it to override 'media' from the content menu in the backend but it doesn't work for me there!

Maybe I'm missing something obvious?

**Comment by dmitriitux** [comment]
in jce, you cannot redefine its button. We need to add a new button from quantum to the JCE editor. In the next version of quantum there will be an option to turn on the front.

**Comment by dmitriitux** [comment]
and redefining media in the admin panel? and in which component?

**Reply by buttonsbond** [reply]
Hi Dimitri, would be good in next version if works in editing at the front end as don't always want users logging into administrator interface.

Not sure what you mean by 'and redefining media.....' - I was referring to the Media menu option and I thought I read in the documentation that you could have quantum pop up instead of the standard media manager.

Thanks for taking the time to reply, and so quickly too.

**Reply by dmitriitux** [reply]
And, in the next version there will be front-loading. The version is already ready, we are still testing it
